.github/workflows/test.yml

```diff
@@ -73,7 +80,10 @@ jobs:
       - name: Setup Terraform ${{ matrix.terraform }}
         uses: hashicorp/setup-terraform@a1502cd9e758c50496cc9ac5308c4843bcd56d36 # v3.0.0
         with:
-          terraform_version: ${{ matrix.terraform }}.*
+          # TODO: Temporary change has been made to `vars.TF_VERSIONS_PROTOCOL_V5` to include the `.*` to enable us
+          # to utilize the alpha/beta versions of Terraform. This should be reverted once Terraform v1.8.0 is released.
+          # terraform_version: ${{ matrix.terraform }}.*
+          terraform_version: ${{ matrix.terraform }}
```

Comment:
Updated to:

`["0.12.*","0.13.*","0.14.*","0.15.*","1.0.*","1.1.*","1.2.*","1.3.*","1.4.*","1.5.*","1.6.*", "1.7.*", "1.8.0-alpha20240228"]`

Will revert in a future PR 😃

Reply:
Successful tests with new `1.8.0-beta1` in `TF_VERSIONS_PROTOCOL_V5`

9c98a52dcaf6051daafd403f055b13ece5efa603 ⚡ 

https://github.com/hashicorp/terraform-provider-local/actions/runs/8177950751/job/22360813566?pr=285